src/main/java/project/bookstore/repository/BookSpecificationBuilder.java

```diff
@@ -0,0 +1,34 @@
+package project.bookstore.repository;
```

Comment:
```suggestion
package project.bookstore.repository.filter;
```

it's better to replace all searching logic to filter package